sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/aggregate/GeneralAggregateFunc.java

```diff
@@ -41,6 +41,7 @@
  *  <li><pre>REGR_R2(input1, input2)</pre> Since 3.4.0</li>
  *  <li><pre>REGR_SLOPE(input1, input2)</pre> Since 3.4.0</li>
  *  <li><pre>REGR_SXY(input1, input2)</pre> Since 3.4.0</li>
+ *  <li><pre>MODE(input1[, inverse])</pre> Since 4.0.0</li>
```

Comment:
This is very hacky. It's kind of we re-define the SQL semantic of the second parameter for MODE function only. Do we have existing examples?

Reply:
Got it. Let's improve it.